### PR TITLE
render hotfix

### PR DESCRIPTION
**Pull request by moruzerinho6**
Now displays author from the message and not from who executed the command.
Fixed attachments being matrix and not an array.

### CODE REVIEW

**Comment by jaxxibae** [comment]
Is it supposed to display the executor's name & avatar? 
image

**Comment by moruzerinho6** [comment]
Yes it's, please talk with me about what message you're trying to render in Switchblade's discord.

**Comment by pedrofracassi** [comment]
Is this RFR?

**Comment by moruzerinho6** [comment]
Waiting for @jaxxibae to give me the message that didn't display the author

**Comment by jaxxibae** [comment]
Ahh yes. https://discordapp.com/channels/445203868624748555/516243264635011072/756607065757057092

File: src/commands/utility/render.js [hunk 1]
```diff
@@ -38,7 +38,10 @@ module.exports = class Render extends Command {
     try {
       if (messageHasNoEmbed) {
         const embed = new SwitchbladeEmbed(author)
-          .setAuthor(author.username, author.displayAvatarURL({ dynamic: true }))
+        embed.setAuthor(link.author.username, link.author.displayAvatarURL({ dynamic: true }))
```
Comment (almeidx) [inline src/commands/utility/render.js:41]:
```suggestion
          .setAuthor(link.author.username, link.author.displayAvatarURL({ dynamic: true }))
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**